lua/lean/infoview.lua

```diff
@@ -35,6 +35,9 @@ local options = {
     mappings = {
       ['K'] = 'click',
       ['<CR>'] = 'click',
+      ['gd'] = 'go_to_def',
+      ['gD'] = 'go_to_decl',
+      ['gt'] = 'go_to_type',
```

Comment:
`gt` is a somewhat useful existing mapping ([for `:tabnext`](https://neovim.io/doc/user/tabpage.html#gt)). `gy` is unused and maybe a good alternative if we don't want to relegate this to `<localleader>`.